mixer/cmd/bundles.go

```diff
@@ -114,10 +114,55 @@ var bundleListCmd = &cobra.Command{
 	},
 }
 
+// Bundle Edit command ('mixer bundle edit')
+type bundleEditCmdFlags struct {
+	noEdit bool
+	editor string
+	add    bool
+}
+
+var bundleEditFlags bundleEditCmdFlags
+
+var bundleEditCmd = &cobra.Command{
+	Use:   "edit [bundle(s)]",
+	Short: "Edit local and upstream bundles",
+	Long: `Edit local and upstream bundle definition files. This command will locate the
+bundle (looking first in local-bundles, then in upstream-bundles), and launch
+an editor to edit it. If the bundle is only found upstream, the bundle file will
+first be copied to your local-bundles directory for editing. When the editor
+closes, the bundle file is then parsed for validity.
+
+The editor is configured in builder.conf via the BUNDLE_EDITOR field, or by flag
+override. Valid options include: nano, vim, emacs, joe, and ctags. The default
```

Comment:
ctags is a valid editor?

Reply:
Haha, is it not? Basic ignorance issue here. As mentioned above, I pulled the list from what packages are installed by `editors`, but I obviously should have done a basic google. Ctags is a package used by one of the actual editors?

I'll pull it.

Reply:
ctags produces a 'tags' database so you can find definitions and uses of variables in C projects.